api/src/main/java/org/openmrs/module/appointments/model/Appointment.java

```diff
@@ -23,6 +23,12 @@ public class Appointment extends BaseOpenmrsData implements Serializable {
     private Integer appointmentId;
     private String appointmentNumber;
     private Date dateCreated;
+    /**
+     * The date appointment was scheduled. This can be today or in the past (for retrospective entry), but not in the future.
```

Comment:
```suggestion
     * The date when the appointment was scheduled. This can be today or in the past (for retrospective entry), but not in the future.
```